build.psm1

```diff
@@ -446,6 +446,32 @@ Fix steps:
         Pop-Location
```

Comment:
@SteveL-MSFT, your last commit had 3 failures in `PowerShell-CI-macos`
Default enablement of Experimental Features.On preview builds, Experimental Features are enabled
```powershell
Expected: Feature PSCommandNotFoundSuggestion to not be Enabled
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/s/test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1: line 45
45:             $expFeature.Enabled | Should -BeEnabled -Name $expFeature.Name
```
Verify approved aliases list.All approved Cmdlets present (no new Cmdlets added, no Cmdlets removed)
```powershell
Expected $null or empty, but got @(@{InputObject=New-PSBreakpoint; SideIndicator=<=}).
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 540
540:         $result | Should -BeNullOrEmpty
```
Verify approved aliases list.All present Cmdlets should have the correct ConfirmImpact
```powershell
Expected $null or empty, but got @(@{Name=Write-Warning; ConfirmImpact=None; SideIndicator=<=}).
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 564
564:         $result | Should -BeNullOrEmpty
```